paasta_tools/cli/cmds/spark_run.py

```diff
@@ -499,29 +466,6 @@ def sanitize_container_name(container_name):
     return re.sub("[^a-zA-Z0-9_.-]", "_", re.sub("^[^a-zA-Z0-9]+", "", container_name))
 
 
-def generate_pod_template_path():
```

Comment:
Where is this moved to?

Reply:
This has been moved to service_configuration_lib